api/src/opentrons/protocol_engine/commands/comment.py

```diff
@@ -0,0 +1,52 @@
+"""Wait for resume command request, result, and implementation models."""
+from __future__ import annotations
+from pydantic import BaseModel, Field
+from typing import Optional, Type
+from typing_extensions import Literal
+
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+
+CommentCommandType = Literal["comment"]
+
+
+class CommentParams(BaseModel):
+    """Payload required to annotate execution with a comment."""
+
+    message: str = Field(
+        None,
+        description="A user-facing message",
+    )
```

Comment:
Oops, my bad, this should be:

```suggestion
    message: str = Field(
        ...,
        description="A user-facing message",
    )
```

Defaulting to `None` only made sense when this was an `Optional[str]`. I'm actually surprised nothing is raising an error because of this. But I think it's causing the generated command schema to be a little funky: `message` isn't listed as `required`.